prime-dotnet-webapi/ApiDbContext.cs

```diff
@@ -42,12 +42,21 @@ public class ApiDbContext : DbContext
     {
         private readonly IHttpContextAccessor _context;
 
+        private readonly bool _isInMemory;
+
         public ApiDbContext(
             DbContextOptions<ApiDbContext> options,
-            IHttpContextAccessor context
+            IHttpContextAccessor context,
+            bool isInMemory = false
             ) : base(options)
         {
             _context = context;
+
+            if (!isInMemory)
```

Comment:
Is there another way to include this in the application?  I'm assuming this is something needed for tests, but it seems odd to have this in the ApiDbContext of the PRIME API.  Not sure of the solution or if there is one, but maybe @james-hollinger can comment.